From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 13 Jan 2016 10:25:05 +0100
Subject: [PATCH] {BUG since 4.5-rc1} ath9k: fix -Wignored-qualifiers

/home/nicolas/projects/linux-patches/linux/drivers/net/wireless/ath/ath9k/channel.c:229:18: error: type qualifiers ignored on function return type [-Werror=ignored-qualifiers]
 static const u32 chanctx_event_delta(struct ath_softc *sc)
                  ^
---
 drivers/net/wireless/ath/ath9k/channel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath9k/channel.c b/drivers/net/wireless/ath/ath9k/channel.c
index 50e614b915f1..7c452af6da39 100644
--- a/drivers/net/wireless/ath/ath9k/channel.c
+++ b/drivers/net/wireless/ath/ath9k/channel.c
@@ -226,7 +226,7 @@ static const char *chanctx_state_string(enum ath_chanctx_state state)
 	}
 }
 
-static const u32 chanctx_event_delta(struct ath_softc *sc)
+static u32 chanctx_event_delta(struct ath_softc *sc)
 {
 	u64 ms;
 	struct timespec ts, *old;
-- 
